packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -60,6 +51,15 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
     ///         This may be removed in the future.
     bool public paused;
 
+    /// @notice Address of the L2OutputOracle contract.
+    L2OutputOracle public L2_ORACLE;
+
+    /// @notice Address of the SystemConfig contract.
+    SystemConfig public SYSTEM_CONFIG;
+
+    /// @notice Address that has the ability to pause and unpause withdrawals.
+    address public GUARDIAN;
```

Comment:
These values are mostly used internally within the Bedrock system. Is there no way that we can use the proper casing on them now?

If not, at some point I think we'll need to do a major release that resets this.

Reply:
This is a good question, I don't love the idea of adding a bunch of extra getters and being stuck with both sets always, then if we change the calls that our contracts make we need to be sure that we upgrade in the correct order or in a single tx so that it always works. Adding them now and then removing the old ones in the future seems like it could work, wondering if we will always be doing this dance of adding and removing. I'm sure that offchain apps may call these values